content/index.pl.md

```diff
@@ -0,0 +1,252 @@
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+## Dostępność jest ważna dla osób fizycznych, firm i społeczeństwa {#important}
+
+Internet staje się coraz bardziej istotnym zasobem dla wielu wymiarów życia: edukacji, zatrudnienia, administracji publicznej, biznesu, opieki zdrowotnej, rekreacji i&nbsp;innych. Dostępność Internetu jest niezbędna, aby zapewnić równy dostęp i&nbsp;równe szanse osobom o&nbsp;różnorodnych możliwościach. Dostęp do informacji i komunikacji międzyludzkiej, w tym Internetu, zdefiniowana jest jako podstawowe prawo człowieka w Konwencji Narodów Zjednoczonych o Prawach osób z niepełnosprawnościami (UN [CRPD](https://www.un.org/development/desa/disabilities/convention-on-the-rights-of-persons-with-disabilities.html)).
+
+Internet oferuje możliwość bezprecedensowego dostępu do informacji i&nbsp;interakcji wielu osobom z&nbsp;niepełnosprawnościami. Oznacza to, że bariery w dostępie do mediów drukowanych, dźwiękowych i&nbsp;wizualnych można znacznie łatwiej pokonać za pomocą technologii internetowych.
+
+Dostępność wspiera włączenie społeczne osób z niepełnosprawnościami, a także innych osób, na&nbsp;przykład:
+
+-   ludzi starszych
+-   ludzi na obszarach wiejskich
+-   ludzi w krajach rozwijających się.
+
+Dostępność internetowa jest w&nbsp;wielu sytuacjach **wymagana przez prawo**.
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Więcej informacji o znaczeniu dostępności" class="simple aside" %}
+{:/}
+
+-   Ogólne informacje o korzyściach dla przedsiębiorstw i przedsięwzięć znajdują się w dokumencie [[Biznesowe uzasadnienie dostępności cyfrowej]](/business-case/).
+    -   Przykłady korzyści płynących z [tworzenia dostępnych mediów audio i&nbsp;wideo](/media/av/) przedstawiono w&nbsp;części [Korzyści dla organizacji](/media/av/users-orgs/#benefits).
+-   Wskazówki dla ustalania wymogów prawnych znajdują się w&nbsp;zarchiwizowanym opracowaniu [Czynniki prawne i&nbsp;polityczne](https://www.w3.org/WAI/business-case/archive/pol).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+## Tworzyć dostępny Internet {#making}
+
+Dostępność Internetu zależy od kilku współpracujących ze sobą komponentów, w tym technologii internetowych, przeglądarek internetowych i innych „programów użytkownika”, narzędzi do tworzenia treści i&nbsp;witryn internetowych.
+
+Inicjatywa na Rzecz Dostępności Internetu W3C ([WAI](/about/participating/)) opracowuje specyfikacje techniczne, wytyczne, techniki i&nbsp;materiały pomocnicze opisujące rozwiązania w zakresie dostępności. Są one uznawane za międzynarodowe standardy dostępności internetowej; na przykład <abbr title="Wytyczne dla dostępności treści internetowej (WCAG)">WCAG 2.0 są także normą <abbr title="International Organization for Standardization">ISO</abbr>: ISO/<abbr title="International Electrotechnical Commission">IEC</abbr> 40500.
```

Comment:
@zwiastunsw The first <abbr> is missing a closing tag.

Reply:
```suggestion
Inicjatywa na Rzecz Dostępności Internetu W3C ([WAI](/about/participating/)) opracowuje specyfikacje techniczne, wytyczne, techniki i&nbsp;materiały pomocnicze opisujące rozwiązania w zakresie dostępności. Są one uznawane za międzynarodowe standardy dostępności internetowej; na przykład <abbr title="Wytyczne dla dostępności treści internetowej (WCAG)">WCAG</abbr> 2.0 są także normą <abbr title="International Organization for Standardization">ISO</abbr>: ISO/<abbr title="International Electrotechnical Commission">IEC</abbr> 40500.
```